RELEASE_NOTES.md

```diff
@@ -0,0 +1,59 @@
+
+# # v20230201002.0.0
+
+## Notes
+
+ The Atlas Golang SDK official v20230201002.0.0 release. 
+
+## Breaking Changes
+
+1. All golang structures starting from `admin.Nullable` have been removed.
+Structures weren't part of the API and should not be used by end users.
+2. Data structures now use golang base types instead of `NullableString` or `NullableInt`.
+For example:
+
+`ThridPartyIntegration.ChannelName NullableString` => `ChannelName *string`
+
+Following list of the models have been affected:
```

Comment:
This change affects the following models:

Reply:
```suggestion
This change affects the following models:
```